adapters/smaato/smaato.go

```diff
@@ -160,24 +160,53 @@ func (adapter *adapter) makeIndividualRequests(request *openrtb2.BidRequest) ([]
 	errors := make([]error, 0, len(imps))
 
 	for _, imp := range imps {
-		request.Imp = []openrtb2.Imp{imp}
-		if err := prepareIndividualRequest(request); err != nil {
-			errors = append(errors, err)
-			continue
-		}
-
-		requestData, err := adapter.makeRequest(request)
+		impsByMediaType, err := splitImpressionsByMediaType(&imp)
```

Comment:
okay, Im completely puzzled O_O
you added `bidfloor` to json but I don't see it anywhere in the adapter code.
how does it work?

Reply:
it is already openrtb, so when request with imp[].bidfloor hits auction endpoint we just pass it along

Reply:
gotcha, thx!